src/github.com/couchbase/sync_gateway/base/sequence_clock.go

```diff
@@ -330,8 +330,8 @@ func (c *SyncSequenceClock) GetHashedValue() string {
 }
 
 func (c *SyncSequenceClock) SetHashedValue(value string) {
-	c.lock.RLock()
-	defer c.lock.RUnlock()
+	c.lock.Lock()
```

Comment:
This was just a bug in the first place - all Set operations on SyncSequenceClock should be getting a write lock.